src/test/mir-opt/const_prop/usize_literal_propagation.rs

```diff
@@ -0,0 +1,9 @@
+// EMIT_MIR_FOR_EACH_BIT_WIDTH
+
+// EMIT_MIR rustc.main.ConstProp.diff
+fn main() {
+    unsafe {
+        let x = 0;
+        std::mem::transmute::<usize, &i32>(x);
```

Comment:
Oki doki! <3

> our test suite should not contain UB

Wait, it doesn't? I was wondering about the tests that are meant to panic (the ones that need to be tagged with `unconditional_panic`). Maybe since those panic, that's not considered UB? (since technically panicking is defined behavior right?)

Reply:
(PS: the test's been changed :))

Reply:
yea panicking is fine :) It's just that having a test that does UB is not very useful as hypothetically rustc may do anything to that test

Reply:
I think you forgot to push ^^

Reply:
> I think you forgot to push ^^

What do you see? I thought something was weird because github still showed `let x = 0;` to me, but after going directly to "Files Changed" and pressing F5, it now shows the correct line:

![image](https://user-images.githubusercontent.com/11747623/79997865-0014c000-8488-11ea-9984-f616a9302133.png)

Reply:
I've been having UI consistency problems with Github lately. Like it showing me "this PR has 17 commits" until I click on the "17 commits" thing and it updates to show me the one commit that there now is. It's been kinda weird XD